docs/agent-oriented-approach.md

```diff
@@ -0,0 +1,49 @@
+
+## Conflicting Environment
+
+Another characteristic of multi-agent systems, which is a direct consequence of their decentralisation, is their conflicting nature. As discussed above, the notion of decentralisation extends as far as ownership and control. Therefore, the different components that make up a decentralised system may each be owned by a different entity, designed according to very different principles and standards, with heterogeneous software and hardware, and each with internal objectives that may be fundamentally inconsistent, worst yet contradictory, with those of others. 
+
+<!--As their name suggest, there are more than one agent in a multi-agent environment, each changing the state of the environment to their own liking. Crucially, as a decentralised system, MAS makes no such assumption that agents are owned by the same entity, or designed homogeneously along the same line of standards and principles. In fact, a distinctive characteristic of a multi-agent environment is that it is inhabited by more than one agent, each being owned by a different stakeholder (individual, company, government).-->
+
+As such, a distinctive characteristic of a multi-agent environment is that it is inhabited by more than one agent, as the name suggests :), where each agent may be owned potentially by a different stakeholder (individual, company, government). Since by design each agent represents and looks after the interests of its owner(s), and because different stakeholders may have unaligned, conflicting, or contradictory interests, it is very common that in a multi-agent system, the agents' objectives, values and preferences may also be unaligned, conflicting, or contradictory.
+
+There are practical implications that follow from the above when it comes to designing an agent. For example, it is not rational for an agent to automatically rely on the information it receives from other agents. The information could be incomplete; what is unrevealed may have been deemed private for strategic reasons. The information could be uncertain, the result of an inaccurate prediction. Or it could be an outright lie, due to the adversarial nature of the environment. Therefore there is a degree of uncertainty attached to almost all information an agent receives in a multi-agent system. One could argue for taking a sceptical approach; treating everything as uncertain, unless proved otherwise.  
```

Comment:
Good opportunity to use bullet points instead of a long paragraph:

```suggestion
There are practical implications that follow from the above when it comes to designing an agent. For example, an agent should not automatically rely on the information it receives from other agents because it could be:

* Incomplete: What is unrevealed may have been deemed private for strategic reasons
* Uncertain: The result of an inaccurate prediction
* False: Due to the adversarial nature of the environment

Therefore there is a degree of uncertainty attached to almost all information an agent receives in a multi-agent system. One could argue for taking a skeptical approach; treating everything as uncertain, unless proved otherwise.
```